### ISSUE TITLE
Release v1.28.1+k3s1 isn't marked as `latest`

### ISSUE THREAD

**Issue by aofei** [k3s-io/k3s#8337]
Not sure if this is intentional, but the [release v1.28.1+k3s1](https://github.com/k3s-io/k3s/releases/tag/v1.28.1%2Bk3s1) isn't marked as `latest`. This results in our update script not being able to catch it automatically:

```
K3S_VERSION=$(curl -fsSL -o /dev/null -w %{url_effective} https://github.com/k3s-io/k3s/releases/latest | rev | cut -d / -f 1 | rev)
```

Not a big problem though, just a friendly reminder to you guys.

**Comment by brandond** [comment]
Nope. We don't want it marked as latest, due to the init container issue called out in the release notes. We'll mark v1.28.2 latest, after it is released by upstream next week.
